shrinkeventfile/shrinkeventfile.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+__VERSION__ = '0.1'
+
+import os
+import h5py
+
+SDS = 'SDS'
+
+# each items is a triplet of (from, to)
+links_to_make = []
+
+
+class NoAttrInHDF5FileException(Exception):
+    '''Raise exception when attribute not found in HDF5 File'''
+
+
+def product(shape):
+    total = 1
+    for num in shape:
+        total *= num
+    return total
+
+
+def get_entries(data):
+    result = {}
+    for item in data:
+        nxtype = SDS
+        if u'NX_class' in data[item].attrs.keys():
```

Comment:
Does shrinkeventfile still support python2?

Reply:
I didn't try to since it is deprecated.
We could try (part of the multi-python testing with tox) but do we need to?

Reply:
Then you can change this line to `'NX_class'` (drop the `u`) as python3 strings are unicode already

Reply:
Thanks!